src/core/geometry/geom3/invert.js

```diff
@@ -8,10 +8,12 @@ const fromPolygons = require('./fromPolygons')
  * @example
  * let B = invert(A)
  */
-const invert = geometry => {
-  let flippedpolygons = geometry.polygons.map(polygon => poly3.flip(polygon))
-  return fromPolygons(flippedpolygons)
+const invert = (geometry) => {
+  const flippedpolygons = geometry.polygons.map((polygon) => {
```

Comment:
same question as elsewhere : why the syntax change, the other is perferctly valid